src/Validation/EventHandlerExtensions.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Copyright (c) Andrew Arnott. All rights reserved.
+// Licensed under the Ms-PL license. See LICENSE.txt file in the project root for full license information.
+
+namespace Microsoft
+{
+    using System;
+    using System.Diagnostics.CodeAnalysis;
+
+    using Validation;
+
+    /// <summary>
+    /// Extension methods to make it easier to safely invoke events.
+    /// </summary>
+    [SuppressMessage("Microsoft.Design", "CA1030:UseEventsWhereAppropriate", Justification = "These aren't events.")]
+    public static class EventHandlerExtensions
```

Comment:
I'm not sure why these were added to the vs-validation library to begin with, and with C# nullable syntax now their value isn't worth copying to this library, IMO.

Reply:
You're saying this can be dropped? I thought it was sort of strange they were in here.